ansible/roles/os_zabbix/vars/template_openshift_logging.yml

```diff
@@ -0,0 +1,62 @@
+---
+g_template_logging:
+  name: Template OpenShift Logging
+
+  zitems:
+  - key: openshift.logging.fluentd.running
+    applications:
+    - EFK
+    value_type: int
+  - key: openshift.logging.fluentd.number_pods
+    applications:
+    - EFK
+    value_type: int
+  - key: openshift.logging.fluentd.node_mismatch
+    applications:
+    - EFK
+    value_type: int
+  - key: openshift.logging.fluentd.number_expected_pods
+    applications:
+    - EFK
+    value_type: int
+  - key: openshift.logging.kibana.site_up
+    applications:
+    - EFK
+    value_type: int
+  - key: openshift.logging.elasticsearch.single_master
+    applications:
+    - EFK
+    value_type: int
+  - key: openshift.logging.elasticsearch.all_numbers_registered
+    applications:
+    - EFK
+    value_type: int
```

Comment:
We need to also monitor the disk usage of the elastic search PV. This, IMO, is the most important thing to monitor as it's the thing I'm sure we'll run out of, possibly quickly.